.github/actions/helm-publish/action.yml

```diff
@@ -12,7 +8,9 @@ runs:
   - shell: bash
     run: |
       mkdir -p target/helm
-      gsutil cp gs://helm.linkerd.io/${{ inputs.branch }}/index.yaml target/helm/index-pre.yaml
+      gsutil cp gs://helm.linkerd.io/edge/index.yaml target/helm/index-pre.yaml
+      bin/helm-bump-edge
+      bin/helm-docs
```

Comment:
Since we now bundle this into the release process, are we able to remove the README.md files and helm-docs checks that run regularly? Or what's the value in preserving the generated files in the repo?

Reply:
We just get those files nicely rendered by github when browsing the repo. But they're also at artifacthub, so I'm not profoundly attached to them :man_shrugging: